diagrams/credential.svg

```diff
@@ -1 +1,18 @@
\ No newline at end of file
+<svg version="1.1" viewBox="-5 -5 330 245" xmlns="http://www.w3.org/2000/svg">
+<style><![CDATA[
+	rect {stroke:#000;stroke-width:1uu;}
+]]></style>
+
+<rect fill="#bad" width="320" height="230" rx="25"/>
+
+<text style="font-family:sans-serif;font-size: 1.5em" x="60" y="40">Verifiable Credential</text>
+
+<rect fill="#dab" x="45" y="65" width="240" height="40" rx="4"/>
```

Comment:
These are not centered.  Should be x=(320-240)/2=40?

Reply:
probably. I'll fix...

Reply:
fixed in bd4f6b3